lib/core/reporters/na.js

```diff
@@ -9,7 +9,25 @@ axe.addReporter('na', function(results, options, callback) {
 	}
 
 	var out = helpers.processAggregate(results, options);
+	var orientation =
```

Comment:
We should probably use the standards-compliant object first:

```js
var orientation = screen.orientation || screen.msOrientation || screen.mozOrientation || {}
```

Just incase someone defined a non-standard property of the same name.

Not a big deal either way tho.

Reply:
We could, I just copied the MDN code to grab it https://developer.mozilla.org/en-US/docs/Web/API/Screen/orientation

Reply:
🤷‍♀️ OK, that's fine I suppose